src/style.css

```diff
@@ -1 +1,99 @@
-/* Styles go here */
+
+html {
+  box-sizing: border-box;
+}
+
+* {
+  box-sizing: inherit;
+}
+
+*::after {
+  box-sizing: inherit;
+}
+
+*::before {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.headMenu {
+  display: flex;
+  position: fixed;
+  width: 100%;
+  background-color: white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin: 0 auto;
+}
+
+.wrapper {
+  width: 1200px;
+  margin: 0 auto;
+}
+
+.headerWrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.headerWrapper:last-child {
+  align-items: center;
+}
+
+.imgMoyo {
+  display: flex;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 50px;
+}
+
+.list {
+  display: flex;
+  flex-wrap: wrap;
+  list-style-type: none;
+}
+
+.header__item {
+  margin-right: 20px;
+}
+
+.header__item:last-child {
+  margin-right: 50px;
+}
+
+.nav__list {
+  text-transform: uppercase;
+
+}
+
+.is-active {
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  color: black;
+}
+
+.is-active:hover {
+  color: #00acdc;
```

Comment:
Use CSS variables for repeatable colors.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)